workflow.rst

```diff
@@ -518,11 +519,24 @@ workflow leaves a place::
         public static function getSubscribedEvents(): array
         {
             return [
-                'workflow.blog_publishing.leave' => 'onLeave',
```

Comment:
What about keeping the string version and show the other version too ?

Reply:
Yes, I agree. I've updated this. Thanks.